hardhat.config.ts

```diff
@@ -3,26 +3,26 @@ import { HardhatUserConfig } from 'hardhat/config'
 import '@nomiclabs/hardhat-ethers'
 import '@nomiclabs/hardhat-waffle'
 import '@openzeppelin/hardhat-upgrades'
-import "@nomiclabs/hardhat-etherscan";
```

Comment:
Hmm. Can you leave these changes out to keep the diff clean?
Seems like they've been added by the linter